### DISCUSSION TITLE
Galaxy Brain

### DISCUSSION THREAD

**Discussion by whitemaninhanoi** [community/community#18712] [Other Feature Feedback, Questions, & Ideas] [answered]
How?

**Comment by jorgesandovalm** [comment, marked as answer]
¡Hola! Para el Galaxy Brain Achievement se requiere que alguien marque a lo menos dos respuesta para una contribución realizada por alguien distinto al autor :).

**Comment by github-staff** [comment]
Thanks so much for posting in our Community @rdelavega63. Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Comment by Hazrat-Ali9** [comment]
How do I get a Galaxy Brain badge
Response
